### DISCUSSION TITLE
BUG: edit "button" disappear when size of window is below 1025 px

### DISCUSSION THREAD

**Discussion by rocchisanijl** [GitbookIO/community#593] [0. General bugs and questions] [answered]
I found that "edit" button disappear when the size of windows is below  1025 px 
and i can't found it nowhere else ???

up to 1024
![image](https://github.com/GitbookIO/community/assets/27018079/d084b408-3bec-4815-98ad-fba42b4a2654)

below 1025
![image](https://github.com/GitbookIO/community/assets/27018079/9442b2e7-97a4-47b4-b7a2-9ada4cc27cab)

**Comment by addisonschultz** [comment, marked as answer]
Hi @rocchisanijl - this is expected behavior, as we don't have an edit mode on smaller devices. The breakpoint for this is 1024 as you've seen!

**Reply by rocchisanijl** [reply]
ok thank you understood. nevertheless it seems it is new because by the past i was able to edit on my smartphone but not anymore event by using "version for computer" of google web browser. Is there a way to use it on a smartphone ?

**Reply by addisonschultz** [reply]
I'm not aware of a way to edit on a smartphone, unfortunately
